### ISSUE TITLE
Werteaktualisierung

### ISSUE THREAD

**Issue by fr43nk** [z-master42/solarflow#11]
Hi,

vielen Dank für die vielen Informationen zu Solarflow und HA.
Alles in allem funktioniert es die Daten vom Zendure Server per mqtt ins HA zu laden.
Was mich aber stört und da haben sie vielleicht mehr Erfahrung: Einige Werte aktualisieren sich nicht immer. Zum Beispiel die Leistung der Panels steht auf einen Wert, der schon Stunden alt ist. Dadurch ergibt sich auch bei der Energieberechnung ein Fehler.
Wissen sie da eine Möglichkeit, wie eine Werteaktualisierung getriggert werden kann oder woran das liegen könnte?

Vielen Dank!

**Comment by z-master42** [comment]
Der Wert der einzelnen Panels aktualisiert sich weniger häufig, als der Gesamtwert (Solar Input Power), so zumindest meine Erfahrung. Grundsätzlich aktualisieren sich die Werte eh nur bei einer Änderung.

Was funktionieren könnte ist, bei den jeweiligen Sensoren (sofern sie über YAML angelegt wurden) ein zusätzliches `force_update: true` anzufügen.

**Comment by fr43nk** [comment]
Für die Änderungserkennung habe ich mir nun ein paar template Sensoren gebaut. Dort werte ich die letzte Aktualisierung aus. Wenn der Wert älter als 1h ist, dann wird der auf Null gesetzt. Mal schauen wie sich das bewährt.
```
      value_template: >
        {% set pData = states.sensor.outputpackpower.last_changed %}
        {% set diff = (now() - pData).total_seconds() %}
        {% if diff >= 3600 %}
        {{ int(0) }}
        {% else %}
        {{states('sensor.outputpackpower') | int(0)}}
        {% endif %}
```

**Comment by z-master42** [comment]
Ok. Bei mir hat es gereicht, dass ich den Default-Wert festgelegt habe. In deinem Beispiel das int(0).

**Comment by fr43nk** [comment]
Jo, das brauche ich, falls die Werte ungültig sind, das Problem bei meiner Anlage ist aber, das Werte einfach stehenbleiben. Zum Beispiel die Leistung der Panels und dann ergibt sich durch die Integration natürlich ein falscher Wert für die Energie.
